Source/Core/Core/State.h

```diff
@@ -95,19 +100,19 @@ u64 GetUnixTimeOfSlot(int slot);
 //    If we're in the main CPU thread then they run immediately instead
 //    because some things (like Lua) need them to run immediately.
```

Comment:
I think this comment may be better off changed. How should this be reworded, if at all?